js/bybit.js

```diff
@@ -6143,4 +7315,112 @@ module.exports = class bybit extends Exchange {
+            throw new NotSupported (this.id + ' fetchTradingFee() is not supported for spot market');
+        }
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.privateGetContractV3PrivateAccountFeeRate (this.extend (request, params));
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "OK",
+        //         "result": {
+        //             "list": [
+        //                 {
+        //                     "symbol": "ETHUSDT",
+        //                     "takerFeeRate": "0.0006",
+        //                     "makerFeeRate": "0.0001"
+        //                 }
+        //             ]
+        //         },
+        //         "retExtInfo": {},
+        //         "time": 1658739027301
+        //     }
+        //
+        const result = this.safeValue (response, 'result', {});
+        const fees = this.safeValue (result, 'list', []);
+        const first = this.safeValue (fees, 0, {});
+        return this.parseTradingFee (first);
+    }
+
+    async fetchTradingFees (params = {}) {
+        /**
+         * @method
+         * @name bybit#fetchTradingFees
+         * @description fetch the trading fees for multiple markets
+         * @param {object} params extra parameters specific to the bybit api endpoint
+         * @returns {object} a dictionary of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure} indexed by market symbols
+         */
+        if (this.version !== 'v3') {
+            throw new NotSupported (this.id + ' fetchTradingFees() is only support for v3');
+        }
+        await this.loadMarkets ();
+        let type = undefined;
+        [ type, params ] = this.handleOptionAndParams (params, 'fetchTradingFees', 'type', 'future');
+        if (type === 'spot') {
+            throw new NotSupported (this.id + ' fetchTradingFees() is not supported for spot market');
+        }
+        const response = await this.privateGetContractV3PrivateAccountFeeRate (params);
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "OK",
+        //         "result": {
+        //             "list": [
+        //                 {
+        //                     "symbol": "ETHUSDT",
+        //                     "takerFeeRate": "0.0006",
+        //                     "makerFeeRate": "0.0001"
+        //                 }
+        //             ]
+        //         },
+        //         "retExtInfo": {},
+        //         "time": 1658739027301
+        //     }
+        //
+        let fees = this.safeValue (response, 'result', {});
+        fees = this.safeValue (fees, 'list', []);
+        const result = {};
+        for (let i = 0; i < fees.length; i++) {
+            const fee = this.parseTradingFee (fees[i]);
+            const symbol = fee['symbol'];
+            result[symbol] = fee;
+        }
+        return result;
+    }
```

Comment:
@sc0Vu currently inside `sign`, we set ` headers['Referer'] = brokerId;` in multiple places, which is messy, can we dry it and centralize its manipulation? 
Like at the beginning or the end:

```Javascript
if (POST) {
                    const brokerId = this.safeString (this.options, 'brokerId');
                    if (brokerId !== undefined) {
                        headers['Referer'] = brokerId;
                    }
}

Reply:
Looks like we only add this when trade in derivatives market. Do we want to send this in other market?

Reply:
@sc0Vu Yeah I think it does not hurt send it in every POST